bridge/src/main/scala/xsbt/ConsoleInterface.scala

```diff
@@ -0,0 +1,68 @@
+/* sbt -- Simple Build Tool
+ * Copyright 2008, 2009 Mark Harrah
+ */
+package xsbt
+
+import xsbti.Logger
+import scala.tools.nsc.{ GenericRunnerCommand, Interpreter, InterpreterLoop, ObjectRunner, Settings }
+import scala.tools.nsc.interpreter.InteractiveReader
+import scala.tools.nsc.reporters.Reporter
+import scala.tools.nsc.util.ClassPath
+
+import dotty.tools.dotc.repl.REPL
+import dotty.tools.dotc.repl.REPL.Config
+
+class ConsoleInterface {
+  def commandArguments(
+    args: Array[String],
+    bootClasspathString: String,
+    classpathString: String,
+    log: Logger
+  ): Array[String] = args
+
+  def run(args: Array[String],
+          bootClasspathString: String,
+          classpathString: String,
+          initialCommands: String,
+          cleanupCommands: String,
+          loader: ClassLoader,
```

Comment:
`loader` isn't passed to the REPL, is that a TODO? If yes that should be noted in a comment